devtools/aesh/src/main/java/io/quarkus/cli/commands/ExecuteUtil.java

```diff
@@ -0,0 +1,106 @@
+package io.quarkus.cli.commands;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+import org.aesh.command.invocation.CommandInvocation;
+import org.apache.maven.shared.invoker.DefaultInvocationRequest;
+import org.apache.maven.shared.invoker.DefaultInvoker;
+import org.apache.maven.shared.invoker.InvocationRequest;
+import org.apache.maven.shared.invoker.InvocationResult;
+import org.apache.maven.shared.invoker.Invoker;
+import org.apache.maven.shared.invoker.MavenInvocationException;
+
+public class ExecuteUtil {
+
+    public static void executeGradle(File projectDirectory, CommandInvocation invocation, String buildTarget) {
+        String gradleExecutable = findExecutable("gradle");
+        if (gradleExecutable == null) {
+            invocation.println("unable to find the gradle executable, is it in your path?");
+        } else {
+            gradleExecutable += File.separator + "bin" + File.separator + "gradle";
+
+            try {
+                Process process = new ProcessBuilder()
+                        .command(gradleExecutable, buildTarget)
+                        .directory(projectDirectory)
+                        .start();
+
+                BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
+
+                String line;
+                while ((line = reader.readLine()) != null) {
+                    invocation.println(line);
+                }
+
+                int exit = process.waitFor();
+                if (exit != 0)
+                    invocation.println("Build failed.");
+            } catch (IOException | InterruptedException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    public static void executeMaven(File projectDirectory, CommandInvocation invocation, String buildTarget) {
+        String mvnPath = findExecutable("mvn");
```

Comment:
this would be ignoring ./mvnw / maven wrapper

Reply:
There is a check first to find out if there is a wrapper present or not.